### PR TITLE
Fix rdfConvertor does not obey interface

### PR DESCRIPTION
**Pull request by renevds**
finishes what was started in #58

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
     "pug": "^2.0.4",
     "tmp": "^0.1.0",
     "winston": "^3.2.1",
-    "yaml": "^1.6.0"
+    "yaml": "^1.6.0",
+    "yarn": "^1.22.11"
```
Comment (pheyvaer) [inline package.json:39]: Why is this added?

File: lib/converters/rdf-converter.js [hunk 2]
```diff
@@ -28,16 +28,16 @@ class RdfConverter extends Converter {
    * @param data - the JSON data to be reformatted
    * @param graphQLLD - GraphQLLD info object (see walder/lib/parsers/graphQLLDParser.js)
    */
-  static convert(format, data, graphQLLD) {
+   convert(format, data, graphQLLD) {
     assert(Object.values(RDF_TYPES).includes(format), 'Unsupported RDF format requested!');
 
     // Always convert to JSON-LD first
-    const jsonld = this._toJSONLD(data, graphQLLD);
+    const jsonld = RdfConverter._toJSONLD(data, graphQLLD);
```
Comment (pheyvaer) [inline lib/converters/rdf-converter.js:35]: Does `_toJSONLD` actually have to stay static in this case then?

File: package.json (hunk 1 above)
Comment (renevds) [reply package.json:39]: This slipped trough somehow, will remove :)

File: lib/converters/rdf-converter.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ class RdfConverter extends Converter {
     if (format === RDF_TYPES.JSON_LD) {
       return jsonld;
     } else {
-      return this._reformat(jsonld, format);
+      return RdfConverter._reformat(jsonld, format);
```
Comment (pheyvaer) [inline lib/converters/rdf-converter.js:40]: Should `_reformat` be static still?

File: lib/converters/rdf-converter.js [hunk 4]
```diff
@@ -56,7 +56,7 @@ class RdfConverter extends Converter {
     // concat arrays
     const concatData = [].concat(...Object.values(data));
 
-    this._structureData(concatData);
+    RdfConverter._structureData(concatData);
```
Comment (pheyvaer) [inline lib/converters/rdf-converter.js:59]: Should `_structureData` be static still?
